### ISSUE TITLE
No prompt when selecting "Open File" or "Open Folder" within VSCode on Linux arm64 ubuntu using proot

### ISSUE THREAD

**Issue by elesbb** [microsoft/vscode#228965]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.93.1
- OS Version: Ubuntu 22.04 LTS arm64

Steps to Reproduce:

1. Install VSCode for arm64 Linux using the provided .deb package
2. Launch VSCode in linux
3. Go to File->Open File or Ctrl+O or File->Open Folder
4. There is no prompt.

**Comment by elesbb** [comment]
@bpasero @deepak1556 

Any update on this? I can provide any logs if needed

**Comment by bpasero** [comment]
Is the dialog not opening at all or opening behind the main window?

Can you please follow the steps in https://github.com/Microsoft/vscode/wiki/Native-Crash-Issues to get at more details around the crash and attach the result here? Thanks!

**Comment by elesbb** [comment]
> Is the dialog not opening at all or opening behind the main window?
I posted on Reddit and someone mentioned this as well. So I minimized VSCode and it would appear the dialog is **not opening at all**
> Can you please follow the steps in https://github.com/Microsoft/vscode/wiki/Native-Crash-Issues to get at more details around the crash and attach the result here? Thanks!
I tried to follow the steps, but no crash log was generated. My guess is simply because it _doesn't crash_ lol. It just does nothing. It still runs and everything is fine, just without the file open/folder open dialog showing

Is there a way to put it into verbose logging mode? Or something similar?

**Comment by bpasero** [comment]
Please see if you can find any logged output that could help us find the source of the issue:
* run `code --verbose` from the command line
* reproduce the issue
* attach the log files you find from running the command "Open Logs Folder"

Thanks!

**Comment by deepak1556** [comment]
Sorry for the delay here,  could you also try our insiders version https://code.visualstudio.com/insiders and confirm if the issue is present.

**Comment by elesbb** [comment]
@deepak1556 @bpasero 

Hey friends!

Thanks for the --verbose switch. Can't believe I didn't try that myself lol.
Some context - I am using Ubuntu ARM64 inside an andronix (proot) environment.

After running code --verbose, I was able to see the logs, the error comes from this: 
`[18937:1001/202935.046812:ERROR:select_file_dialog_linux_portal.cc(769)] Portal returned error: org.freedesktop.DBus.Error.AccessDenied: Portal operation not allowed: Unable to open /proc/18937/root`

Which I have been having other issues with /proc stuff in this build.

I have been using andronix cause I found it first, this lead me to discover `proot-distro` which seems to fix the /proc issues.

I tested out an installation of ubuntu under proot-distro, and vscode successfully shows the open file dialog.

Also, weirdly, the error was output to the terminal, instead of the log files. So I redirected all output to a file instead. Here it is:
[vscode.log](https://github.com/user-attachments/files/17218419/vscode.log)

I guess this can be marked solved then. Not sure if its worth updating vscode to be able to open dialogs without the /proc accessor. Which, maybe it is cause all other programs can open files like geany, mousepad, etc. Its up to you. But I think I will go ahead and get proot-distro setup for myself. Which ultimately fixes this issue
